src/chainparams.cpp

```diff
@@ -168,10 +168,10 @@ class CMainParams : public CChainParams {
         };
 
         chainTxData = ChainTxData{
-            // Data from RPC: getchaintxstats 4096 000000000000000000052d314a259755ca65944e68df6b12a067ea8f1f5a7091
-            /* nTime    */ 1645542140,
-            /* nTxCount */ 712531200,
-            /* dTxRate  */ 2.891036496010309,
+            // Data from RPC: getchaintxstats 4096 00000000000000000009c97098b5295f7e5f183ac811fb5d1534040adb93cabd
+            /* nTime    */ 1661697692,
```

Comment:
```suggestion
            . nTime    = 1661697692,
```

nit: I think you can use designated initializers now

Reply:
done